infoenergia_api/api/reports/reports.py

```diff
@@ -19,7 +19,7 @@ class ReportsView(PaginationLinksMixin, HTTPMethodView):
 
     async def post(self, request, user):
         logger.info("Uploading contracts")
-        report_ids, month = await get_report_ids(request)
+        report_ids, month, report_type = await get_report_ids(request)
```

Comment:
I'm reading get_reports_ids function and I find weird its behavior. I understand that the main idea of the function is to get the ids of the contracts that came in the requests and save it in redis for a later treatment. 
But, first delete a `key:reports` key for a later assignment and read from redis. In total, we access 3 times to redis :scream_cat: 

Can we check together this function, we will find a better solution :smile: